databricks/koalas/generic.py

```diff
@@ -39,6 +39,7 @@ class _Frame(object):
     def to_numpy(self):
         """
         A NumPy ndarray representing the values in this DataFrame
+
```

Comment:
i've rewritten this doc in https://github.com/databricks/koalas/pull/273

Maybe merge 273 first if it looks good? Otherwise there will be a conflict.

Reply:
yup